app/models/piece.rb

```diff
@@ -4,46 +4,52 @@ class Piece < ApplicationRecord
 
   #needs models that inherit from piece called pawn, rook, knight, bishop, queen, king
 
+  def outside_board?(position_x, position_y)
+    if position_x < 0 || position_x > 7 || position_y < 0 || position_y > 7
```

Comment:
This method can be just one line without `if/else`.. 
```
def
 a || b || c || d
end
```

the condition will evaluate to true and false and be returned as in ruby the last line of the method always gets returned :)